src/main/java/wolox/training/models/User.java

```diff
@@ -39,6 +40,9 @@
     @Column(nullable = false)
     private LocalDate birthdate;
 
+    @Column(nullable = false)
+    private String password;
```

Comment:
Use this annotation too: `@JsonProperty(access = Access.WRITE_ONLY)` so that it can't be read